include/functions.h

```diff
@@ -432,6 +432,9 @@ s32 dead_get_variable(ScriptInstance* script, Bytecode var);
 f32 dead_get_float_variable(ScriptInstance* script, Bytecode var);
 s32 dead_set_variable(ScriptInstance* script, Bytecode var, s32 value);
 f32 dead_set_float_variable(ScriptInstance* script, Bytecode var, f32 value);
+f32 func_8002AE14(f32 theta);
```

Comment:
All of these dead functions should go to the bottom of this file where the others are (including DeadUnkNpcAIFunc1)

Reply:
Sorry if I am not understanding your request, but this is all below `Dead Functions` at the bottom of the file already - Or did you mean something different?
![grafik](https://user-images.githubusercontent.com/31699028/127857819-ff6825d4-5673-4211-9633-b142cd19197f.png)

Reply:
oh lol whoops. don't know how I missed that. nevermind :p